.github/workflows/e2e.yml

```diff
@@ -31,12 +31,22 @@ jobs:
       - name: Install Dependencies
         if: steps.deps-cache.outputs['cache-hit'] != 'true'
         run: yarn install
+
+      - name: Build SDK
+        run: yarn build
+      - name: Package SDK
+        run: yalc publish
+      - name: Prepare sample for testing
+        working-directory: ./sample
+        run: sh ./scripts/prepareConfigsForTesting.sh
+      - name: Install SDK in sample
+        working-directory: ./sample
+        run: yalc add @sentry/react-native
```

Comment:
Think its unavoidable now, but lets please split up the CI changes from the implementation so that things are easier to revert/`git blame` in the future.

Reply:
agreed, but splitting up would break the e2e since the CI changes are needed, let's keep this in mind for the next PR but I'd rather merge it for now since it's an important fix.